modules/enterprise/gui/coregui/src/main/java/org/rhq/coregui/server/gwt/BundleGWTServiceImpl.java

```diff
@@ -71,7 +71,7 @@ public ResourceTypeBundleConfiguration getResourceTypeBundleConfiguration(int co
     public BundleVersion createOrStoreBundleVersionViaURL(String url, String username, String password)
         throws RuntimeException {
         try {
-            BundleVersion results = bundleManager.createBundleVersionViaURL(getSessionSubject(), url, username,
+            BundleVersion results = bundleManager.createOrStoreBundleVersionViaURL(getSessionSubject(), url, username,
```

Comment:
I would like to point out that I modified "createBundleVersionViaURL" (changed to  "createOrStoreBundleVersionViaURL") to match method invocation. Though I'm not sure if is the right thing to do or if we should leave it as "createBundleVersionViaURL".